pkg/capacity/capacity.go

```diff
@@ -99,3 +103,15 @@ func (r *ResourceOracle) Disks() (d Disks, err error) {
 
 	return
 }
+
+//GetHypervisor gets the name of the hypervisor used on the node
+func (r *ResourceOracle) GetHypervisor() (string, error) {
```

Comment:
This funstion should return a slice of string ([]string) since virt-what can output multiple line.